src/components/buxy-table-transactions/buxy-table-transactions.scss

```diff
@@ -0,0 +1,57 @@
+.buxy-table-transactions {
+  border: 1px solid #ddd;
+  border-collapse: collapse;
+  width: 100%;
+  text-align: left;
+  box-sizing: border-box;
+}
+
+.buxy-table-transactions-thead {
+  font-family: $font-table-transactions;
+  font-size: $font-size-table-transactions;
+
+  .buxy-table-transactions-thead__th {
+    border-bottom: 1px solid #ddd;
+    padding-left: 15px;
+  }
+  .buxy-table-transactions-thead__tr {
+    height: 40px;
+  }
+  .buxy-table-transactions-thead__tr:hover {
+    background-color: #f5f5f5;
+  }
+  .buxy-table-transactions-thead__tr:first-child {
+    background-color: #7575a3;

```

Comment:
I think you should use Material Components Theme here to make this color:
add class "mdc-theme--primary-bg" for theme primary background and class "mdc-theme--text-primary-on-primary" for the text of your header.
https://material.io/components/web/catalog/theme/

why? because if we decide to make BUXY with green color, the whole website theme will become green excluding this table header.

Reply:
for first, I'm not sure to be justified to use framework just for correction of one element' bg inside component.
for second, I don't undestand about green color... just noticed that 'dialog for adding transaction' and 'Drawer' headers are close to violet

Reply:
as a variant:
 #751aff or #6600ff close to primary from MC

Reply:
It is just a proposal to give this table the same behavior as other components have. To add styles like font family, font size, font color theme, background color theme, border colors via MDC classes instead of direct styles assigning. It's not just about bg color of header , it's about all of table components.